Firestore/Source/Local/FSTMemoryQueryCache.mm

```diff
@@ -28,6 +29,9 @@ @interface FSTMemoryQueryCache ()
 /** Maps a query to the data about that query. */
 @property(nonatomic, strong, readonly) NSMutableDictionary<FSTQuery *, FSTQueryData *> *queries;
 
+@property(nonatomic, strong, readonly)
```

Comment:
Comment. What are we tracking here and why would it change?

Is it the first seen sequence number per document? The last seen?

When can entries be removed?